### PR TITLE
Minor changes in doc/cephfs/eviction.rst

### PR DESCRIPTION
**Pull request by rishabh-d-dave**

### CODE REVIEW

File: doc/cephfs/eviction.rst [hunk 1]
```diff
@@ -102,7 +102,7 @@ the blacklist:
 
     ceph osd blacklist ls
     # ... identify the address of the client ...
-    ceph osd blacklist rm <address>
+    ceph osd blacklist rm <client-instance>
```
Comment (batrick) [inline doc/cephfs/eviction.rst]: Let's have an example with real output from `blacklist ls` and a real instance name passed to `blacklist rm`.

**Comment by rishabh-d-dave** [comment]
> Please also note in the commit message:
>
> Fixes: https://tracker.ceph.com/issues/23334

Done.

File: doc/cephfs/eviction.rst [hunk 2]
```diff
@@ -102,7 +102,7 @@ the blacklist:
 
     ceph osd blacklist ls
```
Comment (batrick) [inline doc/cephfs/eviction.rst]: Include the output of `blacklist ls` and make sure the address matches what is passed to `blacklist rm`.

Comment (rishabh-d-dave) [reply doc/cephfs/eviction.rst]: Done

**Comment by joscollin** [comment]
Jenkins render docs

**Comment by rishabh-d-dave** [comment]
> Doc render available at http://docs.ceph.com/ceph-prs/20927/

Prepended the commands with a '$'  since commands and their outputs were not distinguishable here.

**Comment by joscollin** [comment]
Jenkins render docs

**Comment by joscollin** [comment]
I don't understand what difference `$` brings in the rendered doc.

**Comment by rishabh-d-dave** [comment]
> I don't understand what difference $ brings in the rendered doc.

'$' makes reading a bit more effortless. Also it aids frequent copying of the command from the doc easy. Otherwise, at least for me, I would have to stare for a second to find out that which 2 lines among the 5 lines are commands. 

'$' are less frequent but not absolutely absent in our docs - http://docs.ceph.com/ceph-prs/20927/cephfs/createfs/.

**Comment by LenzGr** [comment]
FWIW, I too was wondering if we had some documented guideline that defines if and how shell commands should be prepended with a `$` or `#` (depending on the privileges required). I would not mind having a consistent and "official" way of how to accomplish this. 

I personally would be in favor of prepending shell commands with the appropriate prompt.

**Comment by joscollin** [comment]
@LenzGr I agree, but I think it would be a huge cleanup in the doc.

**Comment by LenzGr** [comment]
@joscollin probably, but would this be something that would be accepted as a PR? We probably would have to establish/document the "standard" here, then we could discuss how to consolidate this. 

Unfortunately I could not find any section that outlines any documentation standards in the docs yet apart from what is mentioned in `doc/dev/documenting.rst` - would it make sense to extend this chapter accordingly?

**Comment by joscollin** [comment]
@LenzGr I don't see any problem with the changes in this PR. Anyway, this is closer to what you are talking about: http://docs.ceph.com/docs/master/start/documenting-ceph/#documentation-style-guide

**Pull request merged**